Lib/test/test_largefile.py

```diff
@@ -151,9 +151,23 @@ def test_seekable(self):
                 self.assertTrue(f.seekable())
 
 
+def skip_no_disk_space():
```

Comment:
I would prefer to pass explicitly a path, and call it inside the test, not outside (decorator).

Well, like a regular function :-) `skip_no_disk_space(path, required)`.

Reply:
> I would prefer to pass explicitly a path

Agree. Done.

> and call it inside the test, not outside (decorator).

The check is executed right before running the test (and not on module import). Or at least, I guess that is your concern (it was mine too).